prow/git/v2/remote.go

```diff
@@ -72,24 +73,28 @@ func (f *sshRemoteResolverFactory) PublishRemote(_, repo string) RemoteResolver
 	}
 }
 
-type simpleAuthResolverFactory struct {
-	host     string
+type httpResolverFactory struct {
```

Comment:
Since the point of this is to provide simple auth, why rename?

Reply:
No its not. That is something this can do, but it will also work fine anonymously.